subsystems/DiscordRPC/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
Let's make sure that there is a README in `subsystems/DiscordRPC` to explain how this works and how it is supposed to be used. Maybe we should even have a README in the `subsystems` folder itself to explain the separation. Would be a good place to the diagrams you shared in Discord, too 😉